logging/config/cluster-logging-external.adoc

```diff
@@ -1,28 +1,25 @@
 :context: cluster-logging-external
 [id="cluster-logging-external"]
-= Sending logs to external devices using Fluentd plug-ins
+= Sending logs to external devices using Fluentd Forward plug-ins
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} cluster logging allows you to configure Fluentd plug-ins to send logs to external devices. 
+{product-title} cluster logging allows you to configure the Fluentd *out_forward* plug-in to send logs to external devices. 
 
-* *out_foward*. You can use this plug-in to send logs to an external log aggregator.
-* *fluent-plugin-remote-syslog*. You can use this plug-in to send logs to external syslog server.
+You can use the new xref:../../logging/config/cluster-logging-log-forwarding.adoc#cluster-logging-log-forwarding[log forwarding feature], which can be easier to configure than the plugins. Note that log forwarding is currently in Technology Preview.
 
-[NOTE]
```

Comment:
Was the note block remove to avoid stacking it above the important block?

It almost seems like having both might be okay, since TP status is important enough to call out visibly, I think?

Reply:
There is a new log forwarding feature in 4.3 that is in tech preview and has the TP wording. I mention here that the new log forwarding is TP as I suggested they use it instead. It's better, but TP, btw. 
I changed the wording to _Note that the log forwarding feature is currently in Technology Preview._ Does that help?